test/controllers/unattended_controller_test.rb

```diff
@@ -58,29 +58,21 @@ class UnattendedControllerTest < ActionController::TestCase
 
     test "should render spoof pxelinux for a host" do
       get :host_template, params: { :kind => 'PXELinux', :spoof => @rh_host.ip }, session: set_session_user
-      assert assigns(:initrd)
-      assert assigns(:kernel)
       assert_response :success
     end
 
     test "should render spoof pxegrub for a host" do
       get :host_template, params: { :kind => 'PXEGrub', :spoof => @rh_host.ip }, session: set_session_user
-      assert assigns(:initrd)
-      assert assigns(:kernel)
       assert_response :success
     end
 
     test "should render spoof iPXE for a host" do
       get :host_template, params: { :kind => 'iPXE', :spoof => @rh_host.ip }, session: set_session_user
-      assert assigns(:initrd)
-      assert assigns(:kernel)
       assert_response :success
     end
 
     test "should render spoof gpxe for a host" do
       get :host_template, params: { :kind => 'gPXE', :spoof => @rh_host.ip }, session: set_session_user
-      assert assigns(:initrd)
-      assert assigns(:kernel)
```

Comment:
These asserts look like dropped, let's keep them if there is a different place assigning the variable.

Reply:
Moved to the `BaseVariablesTest`